### PR TITLE
Add folder syntax to replace upcoming content.add deprecation

### PR DESCRIPTION
**Pull request by jyaistMap**
* add code for adding an item using _Folder.add()_ instead of _ContentManger.add()_

### CODE REVIEW

File: samples/03_org_administrators/validate_item_metadata.ipynb [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "\n",
```
Comment (cariashuang0417) [inline samples/03_org_administrators/validate_item_metadata.ipynb:48]: <pre class="ql-syntax" spellcheck="false">Line #1.    from arcgis.gis
</pre><p>An unfinished import line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (cariashuang0417) [inline samples/03_org_administrators/validate_item_metadata.ipynb:48]: <pre class="ql-syntax" spellcheck="false">Line #9.        "snippet":"Report on item attributes from aPI"
</pre><p>little typo in snippet - API</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

**Comment by cariashuang0417** [comment]
The replacement looks good to me, other than 2 little tweaks needed. Will build the local site tomorrow and take a look at the changes. Thanks John!

File: samples/03_org_administrators/validate_item_metadata.ipynb (hunk 1 above)
Comment (jyaistMap) [reply samples/03_org_administrators/validate_item_metadata.ipynb:48]: <p>Fixed!</p>

Comment (jyaistMap) [reply samples/03_org_administrators/validate_item_metadata.ipynb:48]: <p>Fixed!</p>

Comment (jyaistMap) [reply samples/03_org_administrators/validate_item_metadata.ipynb:48]: Fixed!

Comment (jyaistMap) [reply samples/03_org_administrators/validate_item_metadata.ipynb:48]: Fixed!

**Pull request merged**